test/e2e/adminUI/tests/group002Home/uxTestHomeView.js

```diff
@@ -2,107 +2,121 @@ var adminUI = require('../../adminUI');
 
 module.exports = {
 	before: function (browser) {
-		browser
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.signinView.id)
-			.setValue(adminUI.cssSelector.signinView.emailInput, adminUI.login.email)
-			.setValue(adminUI.cssSelector.signinView.passwordInput, adminUI.login.password)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.signinView.submitButton)
-			.pause(browser.globals.defaultPauseTimeout);
+		browser.app = browser.page.app();
+		browser.signinScreen = browser.page.signin();
+		browser.homeScreen = browser.page.home();
+		browser.initialForm = browser.page.initialForm();
+		browser.listScreen = browser.page.list();
+		browser.deleteConfirmation = browser.page.deleteConfirmation();
+
+		browser.app.navigate();
+		browser.app.waitForElementVisible('@signinScreen');
+
+		browser.signinScreen.signin();
+		browser.app.waitForElementVisible('@homeScreen');
 	},
 	after: function (browser) {
-		browser
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.allView.logoutIconLink)
-			.pause(browser.globals.defaultPauseTimeout)
-			.end();
+		browser.app.signout();
+		browser.end();
 	},
-	'Home view should allow clicking a nav menu item such as Access to show the list of items': function (browser) {
-		browser
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.allView.accessMenu)
-			.waitForElementVisible(adminUI.cssSelector.listView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.url(adminUI.url)
-			.waitForElementVisible(adminUI.cssSelector.homeView.id)
-			.pause(browser.globals.defaultPauseTimeout)
-			.click(adminUI.cssSelector.allView.fieldListsMenu)
-			.waitForElementVisible(adminUI.cssSelector.listView.id)
-			.pause(browser.globals.defaultPauseTimeout);
+	'Home view should allow clicking a nav menu item such as Access and Fields to show the list of items': function (browser) {
+		browser.app
+			.navigate()
```

Comment:
you already do the app.navigate and waiting for homeScreen in the before function...no need to do it here?